packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -426,15 +427,15 @@ export function isValidAttestationData(
 
   if (!ssz.phase0.Checkpoint.equals(data.source, justifiedCheckpoint)) return false;
 
-  // Shuffling can't have changed if we're in the first few epochs
```

Comment:
I think we still need this condition, otherwise stateEpoch `n` (could be too big) could include attestations from epoch 1

Reply:
I think that's not the case. e.g. Even if we have stateEpoch `9999` the `findAttesterDependentRoot` is just gonna check `two hopes` from the beacon head. So it can't allow attestations from the epoch 1, unless the two hopes reach to a slot in there.